### PR TITLE
bs4 book

### PR DESCRIPTION
**Pull request by maelle**

### CODE REVIEW

File: _output.yml [hunk 1]
```diff
@@ -1,30 +1,8 @@
...
-        <li><a href="./">rOpenSci Blog Guide for Authors and Editors</a></li>
-      after: |
-        <li><a href="https://github.com/rstudio/bookdown" target="blank">Published with bookdown</a></li>
-    edit:
-      link: https://github.com/ropensci-org/blog-guidance/edit/master/%s
-      text: "Edit this chapter"
-    history:
-      link: https://github.com/ropensci-org/blog-guidance/commits/master/%s
-      text: "Chapter edit history"
-bookdown::pdf_book:
```
Comment (maelle) [inline _output.yml:24]: for some reason we had PDF book stuff here but it wasn't used anyway

**Comment by steffilazerte** [comment]
Cool! So that's where the new R for Data Science look came from ;)

**Comment by maelle** [comment]
:wave: @stefaniebutland

**Review by stefaniebutland — APPROVED** [review]
holy cow. It *is* amazing.

**Pull request merged**